javascript/javascript1/week1/script.js

```diff
@@ -0,0 +1,68 @@
+//Age calculator
+
+let yearOfBirth = 1995
+let yearFuture = 2027
+let age = yearFuture - yearOfBirth
```

Comment:
Use `const `instead of `let`, as you don't change these variables further in code.